docs/Backend_documentation/department_doumentation.md

```diff
@@ -0,0 +1,45 @@
+Department
+----------
+
+Current code:
+@main_blueprint.route("/department/<string:department>")
```

Comment:
when including code blocks you can use https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks. You can also add "python" next to the "```" to highlight python syntax.